run-custom-mao.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -xe
+
+# This script needs an machine-api-operator image as an input parameter
+# For example: ./run-custom-mao.sh quay.io/sdasu/machine-api-operator:sdasu-fix
+export SCRIPTS_DIR=$(pwd)
+export CUSTOM_MAO_BUILD=$1
+
+kubectl get secrets --namespace openshift-machine-api | grep mariadb
+
+# Stop CVO, MAO, machine-api-controller and Metal3 deployments
+oc --config /home/stack/dev-scripts/ocp/auth/kubeconfig scale deployment --replicas=0 cluster-version-operator --namespace openshift-cluster-version
+oc --config /home/stack/dev-scripts/ocp/auth/kubeconfig scale deployment -n openshift-machine-api --replicas=0 machine-api-operator
+oc --config /home/stack/dev-scripts/ocp/auth/kubeconfig scale deployment --replicas=0 machine-api-controllers --namespace openshift-machine-api
+oc --config /home/stack/dev-scripts/ocp/auth/kubeconfig scale deployment --replicas=0 metal3 --namespace openshift-machine-api
```

Comment:
why does metal3 need to be scaled down?  When the MAO restarts, either the deployment is the same and nothing needs to happen, or if it changes, it will get automatically restarted.

Reply:
We are not really changing anything inside Metal3's deployment object. We are just adding a Secret type to the openshift-machine-api's namespace. I am not sure if that will result in the Metal3 being restart automatically. I am trying this on my setup.

Reply:
What I noticed was that the when MAO restarted, it started deploying another Metal3 and the old Metal3 deployment was automatically terminated. So, removing lines that scale the Metal3 and machine-api-controller replicas to 0.